docs/dpg-archboard.md

```diff
@@ -0,0 +1,149 @@
+   be able to accessed as dicts and as models as well
+3. We've added glass breaker `send_request` to all of our clients. Clients can now send an `HttpRequest` directly to the server leveraging our client pipelines.
+4. We've also added easy-to-use for SDK writers customizations, and this the foundation of our grow up story. We are no longer wrapping all of generated code, customizing in place.
+
+### Things Staying the Same
+
+1. Client and Operation group structure stays the same. Client looks exactly regular, has endpoint, has credential.
+
+The rest of the archboard is split into two large chunks. In the first half, Yuchao and Changlong are going to take us through the differences between legacy generated code and DPG generated code, and in the second half I'm going to go over the customization story with Metrics Advisor as my service.
+
+
+--------------------------------------------------------------------------
+
+--------------------------------- Yuchao ---------------------------------
+
+## Developer Experience
+
+### Creating A Client
+
+The end-users experience of creating a client is the same as it was before. We are not compromising on any user input here, this behavior is consistent with Python guidelines and existing user behavior.
+
+### Simple GET Request
+
+The overall flow of all operation calls are largely the same as before. Users must still initialize their client, then either call operations directly on the client, or on operation group attributes.
+
+There are two main things I want to talk about with these basic REST calls.
+
+1. You can see here that we've moved all query and header parameters to keyword-only. This really solves all of our potential versioning issues with parameter ordering. Since path parameters are always required, we are comfortable including them as positional arguments, and we are positioning them based off of their location in the url. The body parameter will always be the last positional argument. Finally, all query and header parameters are keyword-only, they're kind of in Python's options bag. Here ordering doesn't matter.
+2. GET requests will also be the main place where we see the differences with us going from models to just returning raw JSON. We want to be very clear here: Python believes its DPG story is complete without models and just returning raw JSON. **Python users are very comfortable with JSON bodies, and in fact we have gotten numerous issues over the years from customers just asking for raw JSON**. Additionally, we have also invested a lot of effort in making sure the structure of the model inputs and outputs are documented for users.
```

Comment:
`Additionally, we have also invested a lot of effort in making sure the structure of the model inputs and outputs are documented for users.`
Not sure why we need to mention model here.